setup.py

```diff
@@ -44,8 +44,8 @@
     keywords=['hamradio', 'radio', 'maritime', 'yaesu', 'standard horizon', 'hx870', 'hx870e', 'firmware', 'flashing'],
     author='Christiane Ruetten',
     author_email='cr@23bit.net',
-    url='https://github.com/cr/pyhx870',
-    download_url='https://github.com/cr/pyhx870/archive/latest.tar.gz',
+    url='https://github.com/cr/hx870',
+    download_url='https://github.com/cr/hx870/archive/master.tar.gz',
```

Comment:
Good catch! I have a `latest` tag in all my other projects and didn't notice that this link doesn't work without. I wonder where else this is broken...

Reply:
That I can’t tell you. But I’m afraid your repository is called `hx870`, not `pyhx870`, so the links are *still* broken…